week-1/K-functions-parameters/exercise4.js

```diff
@@ -1,9 +1,12 @@
 // Declare your function first - call it "add"
+function add(a, b) {
+  return a + b;
+}
 
 // Call the function and assign to a variable `sum`
-
+var sum = add(13, 124);
 console.log(sum);
 
 // ignore below
 
-module.exports = { add };
+module.exports = { sum };
```

Comment:
You shouldn't change the exports (unless there's a mistake in them and someone tells you to). The tests will probably fail if you change them.